src/test/java/de/tum/in/www1/artemis/exam/StudentExamIntegrationTest.java

```diff
@@ -988,8 +988,6 @@ void testSubmitStudentExam_notInTime() throws Exception {
     @Test
     @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
     void testSubmitStudentExam_differentUser() throws Exception {
-        studentExam1.setSubmitted(false);
-        studentExamRepository.save(studentExam1);
         // Forbidden because user object is wrong
         User student2 = userUtilService.getUserByLogin(TEST_PREFIX + "student2");
         studentExam1.setUser(student2);
```

Comment:
Since the control flow of the respective method in the StudentExamRessource is very simple, we can be certain that the studentExam has not changed when we receive the HttpStatus forbidden and it is not necessary to include an additional check here.